extensions/positron-supervisor/src/KallichoreSession.ts

```diff
@@ -1522,7 +1522,7 @@ export class KallichoreSession implements JupyterLanguageRuntimeSession {
 	 */
 	handleJupyterMessage(data: any) {
 		// Deserialize the message buffers from base64, if any
-		if (data.buffers) {
```

Comment:
This is a not super important but I noticed that every single jupyter message was going into this if statement and replacing the `.buffers` property even when they didn't have any buffers. This is because we're not testing the truthy-ness of the array of data.buffers, we're testing the truthiness of there being the property "buffers" on the data object. Since the buffers is an empty array for almost all messages, we were still going in here and... replacing it with an empty array.